openedx/features/edly/tests/test_utils.py

```diff
@@ -346,3 +348,72 @@ def test_create_learner_link_with_permission_groups(self):
 
         edly_user = create_learner_link_with_permission_groups(edly_user)
         assert edly_user.groups.filter(name=edly_user_group).exists()
+
+    def test_get_current_site_invalid_certificate_context_without_site_configuration(self):
+        """
+        Test method returns correct data without site configuration.
+        """
+        test_default_certificate_html_configurations = {
+            'default': {
+                'accomplishment_class_append': 'accomplishment-certificate',
+                'platform_name': 'fake-platform-name',
+                'logo_src': 'fake-logo-path',
+                'logo_url': 'fake-logo-url',
+                'company_verified_certificate_url': 'fake-verified-certificate-url',
+                'company_privacy_url': 'fake-privacy-url',
+                'company_tos_url': 'fake-tos-url',
+                'company_about_url': 'fake-about-url'
+            }
+        }
+
+        expected_current_site_context_data = test_default_certificate_html_configurations['default']
+        curent_site_context_data = get_current_site_invalid_certificate_context(test_default_certificate_html_configurations)
+        assert expected_current_site_context_data['platform_name'] == curent_site_context_data['platform_name']
+        assert expected_current_site_context_data['company_privacy_url'] == curent_site_context_data['company_privacy_url']
+        assert expected_current_site_context_data['company_tos_url'] == curent_site_context_data['company_tos_url']
+
+    def test_get_current_site_invalid_certificate_context_with_site_configuration(self):
+        """
+        Test method returns correct data with site configuration.
+        """
+        with mock.patch('openedx.features.edly.utils.get_current_site_configuration') as current_site_configuration:
```

Comment:
Had to mock it because we are using `get_site()` without passing `request` as it uses `crum (Current Request User Middleware)`